lib/backend/k8s/conversion.go

```diff
@@ -224,18 +224,16 @@ func (c converter) podToWorkloadEndpoint(pod *kapiv1.Pod) (*model.KVPair, error)
 	profile := fmt.Sprintf("ns.projectcalico.org/%s", pod.ObjectMeta.Namespace)
 	workload := fmt.Sprintf("%s.%s", pod.ObjectMeta.Namespace, pod.ObjectMeta.Name)
 
-	// If the pod isn't ready, we can't parse it.
```

Comment:
Shouldn't we still check `isScheduled()` and `isHostNetworked()` ?

It feels as though `isCalicoPod()` should be a combination of the above two and not include the IP address.

Perhaps with a `isReadyCalicoPod()` to also include the IP address.